css/style.css

```diff
@@ -0,0 +1,322 @@
+    padding: 0;
+    position: relative;
+}
+#food-search input {
+    padding: 0 15px;
+    width: calc(100% - 30px);
+}
+div#restrictions input {
+    padding: 0 35px 0 15px;
+    width: calc(100% - 50px);
+}
+p.alert {
+    font-size: 17px;
+    font-weight: 600;
+    letter-spacing: 0.6px;
+    margin-bottom: 0;
+}
+/* –––––––––– OWL CAROUSELS –––––––––– */
+section#recipes, section#menus {
+    padding: 30px 0;
+    margin-bottom: 0;
+}
+section#content {
+    padding: 50px 0;
+    background: #f1f1f1;
+}
+.owl-nav {
+	height: 0;
+    margin: 0;
+    position: absolute;
+    top: -65px;
+    right: 0;
+    margin-top: 0;
+    color: rgba(0,95,125,0.5);
+}
+.owl-theme .owl-nav [class*=owl-]:hover {
+    background: transparent;
+    color: rgba(0,95,125,0.8);
+    text-decoration: none;
+}
+.owl-carousel .owl-nav .owl-prev:before {
+  font-family: "Font Awesome 5 Pro";
+  content: "\f053";
+  font-size: 20px;
+  position: relative;
+  left: 0px;
+  height: auto;
+  padding: 0 7.5px;
+    margin: 0 7.5px;
+}
+.owl-carousel .owl-nav .owl-next:after {
+  font-family: "Font Awesome 5 Pro";
+  content: "\f054";
+  font-size: 20px;
+  position: relative;
+  right: 0px;
+  height: auto;
+  padding: 0 7.5px;
+    margin: 0 7.5px;
+}
+.owl-stage {
+    display: flex;
+}
+.owl-item {
+    padding-bottom: 15px;
+}
+.card {
+    border-radius: 0 20px;
+    overflow: hidden;
+    height: 100%;
+    margin: 0;
+    box-shadow: 0 10px 14px -10px rgba(0,0,0,0.4);
+}
+.card .card-content {
+    padding: 30px 25px;
+}
+.card-image {
+    height: 180px;
+    background: #f9f9f9;
+}
+.image {
+    content: " ";
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    background-size: cover!important;
+    background-position: center center!important;
+}
+div#icons i {
+    margin-right: 8px;
+}
+div#icons i:nth-of-type(3n - 2) {
+    color: #ff8a65;
+}
+div#icons i:nth-of-type(3n - 1) {
+    color: #005F7D;
+}
+div#icons i:nth-of-type(3n) {
+    color: #00bcd4;
+}
+/* –––––––––– FOOTER –––––––––– */
+footer a.btn-large {
+    font-size: 17px;
+    font-weight: 700;
+    letter-spacing: 0.8px;
+    box-shadow: none!important;
+}
+footer .row {
+    margin-bottom: 0;
+    padding: 100px 0;
+    text-align: center;
+}
+
+
+
+
+
+@media screen and (max-width:992px) {
+
+	section#recipes, section#menus {
+	    padding: 20px 0;
+	}
+	section#content {
+	    padding: 40px 0;
+	}
+	h1 {
+	    font-size: 52px;
+	}
+	p {
+	    font-size: 22px;
+	}
+
+}
+@media screen and (max-width: 600px) {
+
+	h2 {
+	    font-size: 28px;
+	    margin: 0 0 20px;
+	}
+	.owl-nav {
+	    top: -50px;
+	}
+	section#recipes, section#menus {
+	    padding: 12px 0;
+	}
+	section#content {
+	    padding: 30px 0;
+	}
+	footer .row {
+	    padding: 80px 0;
+	}
+	form.row {
+	    display: block;
+	    background: transparent;
+	}
+	#food-search, div#restrictions {
+	    margin-bottom: 10px;
+	}
+	div#hero-content input {
+	    background: #fff;
+	    border-radius: 3px!important;
```

Comment:
you should avoid `!important` at all costs

https://uxengineer.com/css-specificity-avoid-important-css/